omc3/correction/utils_check.py

```diff
@@ -0,0 +1,34 @@
+"""
+Correction Test Utils
+---------------------
+
+Utility functions used by `check_corrections.py` as well as `plotting/plot_checked_corrections.py`.
+"""
+from typing import Dict, Union
+
+import pandas as pd
+
+from generic_parser.entrypoint_parser import EntryPointParameters
+from omc3.plotting.plot_optics_measurements import get_optics_style_params, get_plottfs_style_params
+from omc3.utils import logging_tools
+from tfs import TfsDataFrame
+
+Measurements = Dict[str, Union[TfsDataFrame, pd.DataFrame]]
+
+
+LOG = logging_tools.get_logger(__name__)
+
+
+def get_plotting_style_parameters():
+    """ Parameters related to the style of the plots. """
+    params = EntryPointParameters()
+    params.add_parameter(name="individual_to_input",
+                         action="store_true",
+                         help="Save plots for the individual corrections "
+                              "into the corrections input folders. "
+                              "Otherwise they go with suffix into the output_folders."
+                         )
+    params.update(get_optics_style_params())
+    params.update(get_plottfs_style_params())
+    params["plot_styles"]["default"] = params["plot_styles"]["default"] + ["correction_test"]
+    return params
```

Comment:
I would rather have this file be `omc3.utils.correction`

Reply:
I disagree, as the logic at the moment is: `omc3.utils` -> standalone utils. Everything else is mostly contained within the module itself. And as it is only used there, I think it should live in this module. BUT I am not very happy with the naming. Suggestions welcome.

Reply:
Fair enough.